web/templates/template_landing.html

```diff
@@ -164,7 +164,7 @@ <h1 class="brand-heading">Green Analytics</h1>
 
 
   <h4>Accessing Dashboard</h4>
-  <p> <a href='#'>{{ context.dashboard }}/?token={{ context.site_key }}</a> ,
+  <p> <a href='#'>{{ context.dashboard }}?token={{ context.site_key }}</a> ,
```

Comment:
@philipp-classen : Can you update the protocol to https ? It should be fine now, given that we have https enabled. The best way would be the change the protocol in `.env` file.